StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Intent.swift

```diff
@@ -21,7 +21,7 @@ import UIKit
 enum Intent {
     case paymentIntent(STPPaymentIntent)
     case setupIntent(STPSetupIntent)
-    case deferredIntent
+    case deferredIntent(STPElementsSession)
 
     var clientSecret: String {
```

Comment:
Guessing this will become a `String?` eventually.

Reply:
I'm guessing we'll delete this and make its users explicitly handle all cases
```
switch intent {
case paymentIntent(let pi):
  pi.clientSecret...
case setupIntent(let si):
  si.clientSecret...
case deferredIntent:
  ...ask merchant to create the intent client secret...
```